app/logConfig.py

```diff
@@ -0,0 +1,50 @@
+LOGGING_CONFIG = { 
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': { 
+        'standard_formatter': { 
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        },
+        # 'request_formatter': { 
+        #     'format': '%(asctime)s [%(levelname)s] %(name)s: %(type)s : %(clientip)s : %(endpoint)s : %(message)s'
+        # }
+    },
+    'handlers': { 
+        # 'req_handler': { 
+        #     'level': 'INFO',
+        #     'formatter': 'request_formatter',
+        #     'class': 'logging.handlers.RotatingFileHandler',
+        #     'filename': 'customlogs.log',
+        #     'mode': 'a',
+        #     'maxBytes': 20,
+        #     'backupCount': 0
+        # },
+        'standard_handler':{
+            'level': 'DEBUG',
+            'formatter': 'standard_formatter',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': 'serverlog.log',
+            'mode': 'a',
+            'maxBytes': 1024,
+            'backupCount': 1
+        },
+        'error_handler': { 
+            'level': 'ERROR',
+            'formatter': 'standard_formatter',
+            'class': 'logging.StreamHandler',
+            'stream': 'ext://sys.stdout',  # Default is stderr
```

Comment:
Why not writing ERROR into files? We need to persist to debug, keeping it at console is hard.
We can have both streams and file handler?

Reply:
It is an additional handler to log it into console. The 'standard_handler' will anyway log it into the log file

Reply:
standard_handler level is DEBUG. Will it changed to ERROR in prod? if so write a comment for the same with 'TODO tag' prefixing the comment

Reply:
No. These are 2 handlers. When an error/exception/critical is logged it will be displayed both on console and stored in file